requirements.txt

```diff
@@ -0,0 +1,2 @@
+pyttsx3==2.90
```

Comment:
Hi @paulopacitti I think there are some more dependencies required when using this module for linux  see -> https://pypi.org/project/pyttsx3/ , it would be better to add them as well for linux users.

Reply:
> """Saving Voice to a file""" # On linux make sure that 'espeak' and 'ffmpeg' are installed

I believe that `espeak` and `ffmeg` are not python dependencies, so It could not fit in `requirements.txt`. I'll add to `README.md` instead :)